docs/source/tutorials_source/package/tutorial_moco_memory_bank.py

```diff
@@ -208,52 +215,72 @@
 # .. note:: We use a split batch norm to simulate multi-gpu behaviour. Combined
 #   with the use of batch shuffling, this prevents the model from communicating
 #   through the batch norm layers.
-
 class MocoModel(pl.LightningModule):
     def __init__(self):
         super().__init__()
         
         # create a ResNet backbone and remove the classification head
         resnet = lightly.models.ResNetGenerator('resnet-18', 1, num_splits=8)
-        backbone = nn.Sequential(
+        self.backbone = nn.Sequential(
             *list(resnet.children())[:-1],
             nn.AdaptiveAvgPool2d(1),
         )
 
-        # create a moco based on ResNet
-        self.resnet_moco = \
-            lightly.models.MoCo(backbone, num_ftrs=512, m=0.99, batch_shuffle=True)
+        # create a moco model based on ResNet
+        self.projection_head = MoCoProjectionHead(512, 512, 128)
+        self.backbone_momentum = copy.deepcopy(self.backbone)
+        self.projection_head_momentum = copy.deepcopy(self.projection_head)
+        deactivate_requires_grad(self.backbone_momentum)
+        deactivate_requires_grad(self.projection_head_momentum)
```

Comment:
Nice! This shows that MoCo really has those 4 building blocks and how they belong together.